Units/parser-python.r/newlines-cr.d/README

```diff
@@ -0,0 +1,2 @@
+Tags are right, but location are not recorded correctly.
+Likely due to a bug in main part line number recording with CRs alone.
```

Comment:
In tthat case, could you change the suffix of this directory to `.b` from `.d`?
So we can remember there is a bug.